exercises/sum.cpp

```diff
@@ -1,8 +1,15 @@
-/*
-  Write a program that takes as input two numbers and print the sum.
+//somma!
 
-  Output:
-  Insert the first number: 1
-  Insert the second number: 2
-  Sum: 3
-*/
+#include <iostream>
+
+using namespace std;
+
+int main(){
+    int x, y;
+    cout << "Inserisci il primo numero : " << endl;
+    cin >> x;
+    cout << "Inserisci il secondo numero : "<< endl;
+    cin >> y;
+    cout << x+y;
+    return 0;
```

Comment:
Il return 0 è poco esplicativo di per sè.
Potresti fare un:
`#define EXIT_SUCCESS 0` e inserire alla fine del main `return EXIT_SUCCESS;` in modo tale da incrementare la leggibilità del codice.